sr_system_info/src/sr_system_info/system_info.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2020 Shadow Robot Company Ltd - All Rights Reserved. Proprietary and Confidential.
+# Unauthorized copying of the content in this file, via any medium is strictly prohibited.
+
+import argparse
+import psutil
+import rospy
+import rospkg
+import subprocess
+import yaml
+
+
+class SystemInfo(object):
+
+    def __init__(self):
+        self.rospack = rospkg.RosPack()
```

Comment:
private variables with leading underscore

Reply:
Fixed in https://github.com/shadow-robot/sr_utils/pull/6/commits/321bb546d91af88f074b8164506d2429c7b1d232